### ISSUE TITLE
Add XGBoostClassifier

### ISSUE THREAD

**Issue by sanketsarang** [blobcity/autoai#49]
Add XGBoostClassifier model into the library.

Primary File to Change: https://github.com/blobcity/autoai/blob/main/blobcity/config/classifier_config.py

Reference XGBoostClassifier Implementation:
https://github.com/blobcity/ai-seed/blob/main/Classification/XGBoost/XGBoostClassifier.ipynb

Dependencies if any, must be appropriately added. Test run of the train function on a classification problem must pass, and the function must attempt to train a XGBoostClassifier as a potential best fit model.

**Comment by balamurugan1603** [comment]
Assign me I will do it. We have to directly add the model and its optimal hyperparameters in the models dictionary ryt?

**Comment by sanketsarang** [comment]
> We have to directly add the model and its optimal hyperparameters in the models dictionary ryt?

Yes that is correct.
